Kernel/Graphics/VirtIOGPU/GPU.cpp

```diff
@@ -47,6 +46,12 @@ GPU::GPU(PCI::Address address)
     }
 }
 
+GPU::GPU(PCI::Address address)
+    : VirtIO::Device(address)
+    , m_scratch_space(MM.allocate_contiguous_kernel_region(32 * PAGE_SIZE, "VirtGPU Scratch Space", Memory::Region::Access::ReadWrite))
```

Comment:
Ideally, constructors should not allocate at all.
I know that you're just moving this code around, just making a general comment :)

Reply:
I will fix this, thanks for the comment!

Reply:
This will be fixed in #9818 :)